### PR TITLE
check document module

### PR DESCRIPTION
**Pull request by bealdav**
Nothing todo

### CODE REVIEW

**Comment by bwrsandman** [comment]
Just update `openerp/openupgrade/doc/source/modules70-80.rst` to reflect the coverage of this module.

**Comment by bealdav** [comment]
sorry
done now
thanks for the review

File: addons/document/migrations/8.0.2.1/openupgrade_analysis_WORK.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---Fields in module 'document'---
+ =====>  NOTHING TODO <======
+document     / ir.attachment            / file_type (char)              : module is now 'mail' ('document')
+document     / ir.attachment            / file_type (char)              : now a function
+document     / report.document.user     / user_id (integer)             : relation is now 'res.users' ('False')
+document     / report.document.user     / user_id (integer)             : type is now 'many2one' ('integer')
```
Comment (bwrsandman) [inline addons/document/migrations/8.0.2.1/openupgrade_analysis_WORK.txt:8]: Should the linking be done from the `user_id` integer to the new many2one relation?
Is this already done automatically?

**Comment by bwrsandman** [comment]
:+1:
Glad to see the migration of document is easier in 8.0 than it was in 7.0.
The style changed from markdown to html last year, we had to convert it in the migration script.

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**